src/app/(sidemenu)/w/[workspaceSlug]/crm/contacts/_components/ImportDialog.tsx

```
@@ -0,0 +1,512 @@
+  const handleStartImport = () => {
+    importMutation.mutate({
+      workspaceId,
+      source,
+      dateRange: source === "CALENDAR" || source === "BOTH" ? dateRange : undefined,
+    });
+  };
+
+  // Calculate progress percentage
+  const progressPercentage =
+    importStatus?.totalContacts ?? 0 > 0
+      ? Math.round(
```

Biome (2.1.2)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)